dist/amazon-linux/2/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM amazonlinux:2 AS build
+
+ENV REFRESHED_AT 20220926T210646Z
+ENV LANG=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+ENV MIX_ENV=prod
+
+# Set the right versions
+ENV ERLANG_VERSION 25.1.1
+ENV ELIXIR_VERSION 1.14.1
+ENV ASDF_DATA_DIR /opt/asdf
+ENV ASDF_DIR /opt/asdf
+
+RUN yum install -y ruby-devel git openssl-devel ncurses-devel wget && \
+    yum groupinstall -y  "Development tools" --setopt=group_package_types=mandatory && \
+    amazon-linux-extras install ruby2.6 && gem install fpm && yum clean -y all && rm -rf /var/cache/yum
+
+RUN cd /tmp && \
+  wget "https://github.com/erlang/otp/releases/download/OTP-$ERLANG_VERSION/otp_src_$ERLANG_VERSION.tar.gz" -O otp.tar.gz && \
+  mkdir otp && \
+  tar -xzf otp.tar.gz -C otp --strip-components=1 && \
+  cd otp && \
+  ./configure && make && make install && rm -rf /tmp/otp
+
+RUN cd /tmp && \
+  wget "https://github.com/elixir-lang/elixir/archive/v$ELIXIR_VERSION.tar.gz" -O elixir.tar.gz && \
+  mkdir elixir && \
+  tar -xzf elixir.tar.gz -C elixir --strip-components=1 && \
+  cd elixir && \
+  make && make install && rm -rf /tmp/elixir
```

Comment:
I decided to just build elixir and erlang otp from source because adding a repository entry found here https://www.erlang-solutions.com/downloads/ contains an old version of elixir & erlang